front/app/components/PostCardsComponents/ViewButtons.tsx

```diff
@@ -79,21 +80,26 @@ interface Props {
 }
 
 const ViewButtons = memo<Props>(({ className, selectedView, onClick }) => {
+  const theme = useTheme();
   const isListViewSelected = selectedView === 'card';
   const isMapViewSelected = selectedView === 'map';
   const listButtonRef = useRef<HTMLButtonElement | null>(null);
   const mapButtonRef = useRef<HTMLButtonElement | null>(null);
-  const theme = useTheme();
+
+  const [viewChanged, setViewChanged] = useState<boolean | null>(null);
 
   useEffect(() => {
-    selectedView === 'map'
-      ? mapButtonRef.current?.focus()
-      : listButtonRef.current?.focus();
-  }, [selectedView]);
+    if (viewChanged) {
+      selectedView === 'map'
+        ? mapButtonRef.current?.focus()
```

Comment:
We only want to update this focus if the user clicks/selects the tabs. Otherwise we end up setting focus when the page loads (which leads to the issue we were seeing on production).

We had the viewChanged/setViewChanged behaviour before but it was removed during some cleanup ([commit](https://github.com/CitizenLabDotCo/citizenlab/commit/82afd644a6d4dc5cb415b73e85fddd6878b084ac)). Perhaps there's a better solution, but I've just added this back in for now.